.github/workflows/pre-submit.yaml

```diff
@@ -0,0 +1,36 @@
+name: PreSubmit-Build
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Build OCI image
+    runs-on: ubuntu-latest
+
+    # https://docs.github.com/en/actions/reference/authentication-in-a-workflow
+    permissions:
+      contents: read
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup QEMU
+      uses: docker/setup-qemu-action@v1.2.0
+
+    - uses: chainguard-dev/actions/setup-kind@main
+      with:
+        k8s-version: 1.22.x
+        registry-authority: registry.local:5000
```

Comment:
Worth noting we just do this for the registry?  Might be worth having a separate action for just spinning up a registry, but we can do that separately

Reply:
will do that

Reply:
got
```
Error: failed to build OCI index: failed to publish: Get "https://registry:5000/v2/": http: server gave HTTP response to HTTPS client
2022/03/28 16:18:35 error during command execution: failed to build OCI index: failed to publish: Get "https://registry:5000/v2/": http: server gave HTTP response to HTTPS client
```

using a local registry